blog/content/post/2020-08-05-PCIe-features.md

```diff
@@ -0,0 +1,213 @@
+---
+title: What features PCIe has?
+abstract: 'Introductory blog post to PCIe features.
+          In this article you can read what PCIe capability is
+          and see examples of such capabilities.'
+cover: /covers/PCIExpress.jpg
+author: marek.kasiewicz
+layout: post
+published: true
+date: 2020-08-05
+archives: "2020"
+
+tags:
+  - Firmware
+categories:
+  - Firmware
+
+---
+
+## What this blog is about
+
+Goal of this post is to introduce readers to the concept of PCI Express
+capabilities. It briefly describes what capability is and how it works. It also
+lists some example capability described in PCI Express (PCIe) specification.
+
+## Introduction
+
+PCI Express uses inverted tree topology. It starts from the Root Complex which
+is connected directly to the CPU and RAM. Root Complex branches to Switches and
+Endpoints. Switches branch founder and can be connected to more Switches or to
+Endpoints. Endpoints reside at the bottom of branches and have only one
```

Comment:
too many repetitions: endpoints, switches and so on... maybe again, merge sentences int one bigger, to avoid repetitions?